docs/docs.less

```diff
@@ -2,7 +2,7 @@
  * Styles below are NOT distributed with HelixUI assets *
 \* ==================================================== */
 @import (reference) 'vars';
-@import (reference) 'components/box/index';
+//@import (reference) 'components/box/index';
```

Comment:
Please remove this line rather than commenting it out.

I used comments in the `spec` files to comment out tests, since GitHub treats `_spec.scss` as a new file rather than a move/rename, so it was harder to see the changes.

Since `docs/docs.less` hasn't moved, we can see the changes inline:

<img width="1259" alt="Screen Shot 2020-01-28 at 1 29 51 PM" src="https://user-images.githubusercontent.com/10120600/73298072-513b4d80-41d2-11ea-96c9-69ea09d2a639.png">

Reply:
Changes have been done, please review.

Reply:
Changes have been done, please review.